matchms/Metadata.py

```diff
@@ -195,3 +195,14 @@ def data(self, new_dict):
                 self.harmonize_keys()
         else:
             raise TypeError("Expected input of type dict or PickyDict.")
+    
+    @staticmethod
```

Comment:
What is the expected use case? Is is to remove all the known_key_conversions, or to replace/add just a few? In the last case, it might be better to replace set_key_replacements with change_key_replacements, which should add the given keys to the known_key_conversions , or will overwrite if it was already given in the known_key_conversions.

Reply:
The use case is kind of flexible - can be disabling the key conversions or just setting it to a user defined value - since it is such a minimal and non-invasive change I didn't give that much thought to be fair.